o-fish-ios/Views/PatrolBoat/PatrolSummary/PatrolReportView.swift

```diff
@@ -10,6 +10,7 @@ import SwiftUI
 struct PatrolReportView: View {
 
     var report: ReportViewModel
+    var showingOnSummary = false
```

Comment:
Why is this needed? Only `true` is ever passed to this view

Reply:
We use PatrolReportView on DraftBoardingLoadedStateView, with  false by default, and on PatrolSummaryView with parameter true. So this parameter is needed.

Reply:
Ah - that makes sense – `DraftBoardingLoadedStateView` isn't in this branch and so Xcode couldn't find the reference!